tests/resources/test_git_resource.py

```diff
@@ -67,9 +67,9 @@ class MockCommit():
         mock_variable.called == True
 
         assert len(files) == 2
-        assert files[0][0] == "LICENSE"
+        assert files[0][0] == "LICENSE.html"
```

Comment:
This is not OK to add `.html` to fix the test : a lot of repositories use 'LICENSE' as the conventional name of the license. When browsing a website with a LICENSE, the license file's URL is `https://host/LICENSE` and not `https://host/LICENSE.html`.

Reply:
Fixed